homeassistant/components/sensor/__init__.py

```diff
@@ -66,6 +66,7 @@
 
 DEVICE_CLASSES_SCHEMA: Final = vol.All(vol.Lower, vol.In(DEVICE_CLASSES))
 
+CONF_STATE_CLASS: Final = "state_class"
```

Comment:
Maybe introduce a const.py file instead.

Reply:
Should the move to const.py then be done in a separate PR?

Reply:
Why? It is added here?

Reply:
Can't follow. Do you want to have only conf_state_class in const.py or all constants used in climate/__init__.py?

Reply:
This PR adds a new constant... for global use... right?
I'm suggesting to introduce a const.py file and put this one into that file instead.

But as said below in the review comment, before considering doing that, I would like a sign-off from @MartinHjelmare

Reply:
We normally only define constants for config keys in entity integrations when the constants  are used in the integration itself, like in scene and legacy device_tracker. So I'm a bit hesitant to add it here. If we do add it in the integration I think a `const.py` module is a good idea (option 1). The other option is to add it to the the global `const.py` module since it's a config option that is used by a couple of integrations already (option 2).

I don't have a strong opinion either way. I'd go with option 1, right now.